config/api.php

```diff
@@ -208,7 +208,7 @@
     |
     | Responses can be returned in multiple formats by registering different
     | response formatters. You can also customize an existing response
-    | formatter.
+    | formatter by setting options to configure it's output.
```

Comment:
All of the the instances where you changed this were originally correct.

It's = "it is"
Its = "possessive"